### PR TITLE
Adding simple guide to run puppeteer on Heroku

### PR DESCRIPTION
**Pull request by timleland**

### CODE REVIEW

**Comment by timleland** [comment]
I signed it!

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -155,6 +155,8 @@ how to run this setup from a webserver running on App Engine Flex (Node).
 
 Running Puppeteer on Heroku requires some additional dependencies that aren't included on the Linux box that Heroku spins up for you. To add the dependencies on deploy, add the Puppeteer Heroku buildpack to the list of buildpacks for your app under Settings > Buildpacks.
 
+Simple Guide: https://timleland.com/headless-chrome-on-heroku/
```
Comment (ebidel) [inline docs/troubleshooting.md]: I don't mind additional helpful resources, but can you move this to the bottom of the section? It's in the middle of the step by step explainer.

Maybe:
`There's also another [simple guide](https://timleland.com/headless-chrome-on-heroku/) from @timleland includes a sample project: https://timleland.com/headless-chrome-on-heroku/.`

**Review by ebidel — CHANGES_REQUESTED** [review]

File: docs/troubleshooting.md (hunk 1 above)
Comment (timleland) [reply docs/troubleshooting.md]: Good idea. Made the changes. Thanks

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -155,10 +155,10 @@ how to run this setup from a webserver running on App Engine Flex (Node).
...
 
-Simple Guide: https://timleland.com/headless-chrome-on-heroku/
-
 The url for the buildpack is https://github.com/jontewks/puppeteer-heroku-buildpack
 
 When you click add buildpack, simply paste that url into the input, and click save. On the next deploy, your app will also install the dependencies that Puppeteer needs to run.
 
 If you need to render Chinese, Japanese, or Korean characters you may need to use a buildpack with additional font files like https://github.com/CoffeeAndCode/puppeteer-heroku-buildpack
+
+There's also another [simple guide](https://timleland.com/headless-chrome-on-heroku/) from @timleland includes a sample project: https://timleland.com/headless-chrome-on-heroku/.
```
Comment (ebidel) [inline docs/troubleshooting.md]: "from @timleland includes" -> "from @timleland that includes"

**Review by ebidel — CHANGES_REQUESTED** [review]

**Comment by timleland** [comment]
Good catch 👍

**Review by ebidel — APPROVED** [review]

**Pull request merged**